examples/image_classifier/resnet_152_batch/README.md

```diff
@@ -55,7 +55,7 @@ curl http://127.0.0.1:8080/predictions/resnet152 -T ./serve/examples/image_class
 
     ```bash
 
-    torch-model-archiver --model-name resnet-152-batch --version 1.0  --serialized-file resnet-152-batch.pt --extra-files serve/examples/image_classifier/index_to_name.json  --handler serve/examples/image_classifier/resnet_152_batch/resnet152_handler.py
+    torch-model-archiver --model-name resnet-152-batch --version 1.0  --serialized-file resnet-152-batch.pt --extra-files serve/examples/image_classifier/index_to_name.json  --handler image_classifier
```

Comment:
@shivamshriwas Is the --handler value correct? Shouldn't it be the actual file path for the handler .py file?

Reply:
@chauhang Due to batch support in default handler's [with this PR], resnet152_handler handler became redundant and hence same has been removed.